test/behavior/vector.zig

```diff
@@ -1442,19 +1442,22 @@ test "store vector with memset" {
     if (builtin.zig_backend == .stage2_riscv64) return error.SkipZigTest;
 
     if (builtin.zig_backend == .stage2_llvm) {
+        // LLVM 16 ERROR: "Converting bits to bytes lost precision"
```

Comment:
Note that this test is being re-enabled for all targets in https://github.com/ziglang/zig/pull/21183.